internal/driver/gomobile/canvas.go

```diff
@@ -337,7 +337,7 @@ func (c *mobileCanvas) tapDown(pos fyne.Position, tapID int) {
 
 	co, objPos, layer := c.findObjectAtPositionMatching(pos, func(object fyne.CanvasObject) bool {
 		switch object.(type) {
-		case fyne.Tappable, mobile.Touchable, fyne.Focusable:
+		case mobile.Touchable, fyne.Focusable:
```

Comment:
What is the rationale behind this change? On the face of it, tapDown should be searching for a Tappable object, what am I missing?

Reply:
I can see the confusion here yes. `Tappable` happens on tapUp, the tapDown does not trigger those events, it's inclusion in this list was incorrect because it could find a tappable object above a focusable object, and then not action any event as a result. And so focusable would not get the event it should because it thought tappable overrode.

Reply:
Makes sense, thanks for the explanation